package.json

```diff
@@ -22,19 +22,18 @@
 		"@wordpress/editor": "^14.10",
 		"@wordpress/element": "^6.10",
 		"@wordpress/format-library": "^5.10",
+		"@wordpress/icons": "^10.12.0",
 		"@wordpress/preferences": "^4.10",
-		"@wordpress/private-apis": "^1.10",
-		"react": "^18.3"
```

Comment:
Unnecessary dependency declaration, as the React dependency is declared via `@wordpress/element`.